### ISSUE TITLE
Docker tests fail because installer assumes AWX object exists

### ISSUE THREAD

**Issue by rooftopcellist** [ansible/tower-operator#514]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
Molecule[docker] tests run the installer directly, which fails because the installer role assumes an AWX object is created (because it patches labels onto the AWX object).  

```
TASK [installer : Patching labels to AWX kind] *********************************
fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'meta' is undefined\n\nThe error appears to be in '/home/chadams/awx-operator/roles/installer/tasks/main.yml': line 2, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n---\n- name: Patching labels to AWX kind\n  ^ here\n"}
```

Options:
1. Remove docker tests and just use minikube tests?
2. Change the installer to only try to patch labels when the AWX object exists.  

We probably want to update our Contributing.md docs accordingly.  

##### ENVIRONMENT
* AWX version: 0.13.0
* AWX install method: Molecule tests in docker

##### STEPS TO REPRODUCE

1. Install molecule[docker] from devel (3.4.0 is not compatible with ansible 2.11.4 yet, fixed in devel)
2. `molecule test`

**Comment by shanemcd** [comment]
Yeah, let's just remove that option. Too much maintenance for no good reason.
